### ISSUE TITLE
How to set icon to installer?

### ISSUE THREAD

**Issue by Truth1987** [mherrmann/fbs-tutorial#32]
I am developing an app on windows.
I am already check the tutorial of fbs.
But I still don't know how to set my own icon to app installer.

Thanks

**Comment by mherrmann** [comment]
I'm afraid fbs doesn't yet support changing the icon *of the installer*.

**Comment by Truth1987** [comment]
It's ok.
I am just worried that I am wrong.

**Comment by yash-akbari** [comment]
It's Now Solved , I am using fbs 0.9.0 , i replaced "src/main/icons/Icon.ico" file wtih my icon file.

Note: Don't use icon files whose types where changed manually(by renaming in windows). it gives large traceback.

**Comment by Dorian569** [comment]
Can we customize installer ? Adding a terms and license page for example ?

**Comment by mherrmann** [comment]
Yes, copy https://github.com/mherrmann/fbs/blob/master/fbs/_defaults/src/installer/windows/Installer.nsi into `src/installer/windows/Installer.nsi` and modify.

**Comment by Dorian569** [comment]
It works! Thank you for your help 🙂

________________________________
De : Michael Herrmann ***@***.***>
Envoyé : jeudi 2 mai 2024 11:57
À : mherrmann/fbs-tutorial ***@***.***>
Cc : Dorian CARDOSO ***@***.***>; Comment ***@***.***>
Objet : Re: [mherrmann/fbs-tutorial] How to set icon to installer? (#32)


Yes, copy https://github.com/mherrmann/fbs/blob/master/fbs/_defaults/src/installer/windows/Installer.nsi into src/installer/windows/Installer.nsi and modify.
